cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -109,13 +111,32 @@ func NewCmdApply(apf *applyPlanFlags) *cobra.Command {
 func runApply(flags *applyFlags, userVersion string) error {
 
 	// Start with the basics, verify that the cluster is healthy and get the configuration from the cluster (using the ConfigMap)
-	klog.V(1).Infoln("[upgrade/apply] verifying health of cluster")
+
 	klog.V(1).Infoln("[upgrade/apply] retrieving configuration from cluster")
-	client, versionGetter, cfg, err := enforceRequirements(flags.applyPlanFlags, flags.dryRun, userVersion)
+	client, cfg, versionGetter, err := upgrade.GetUpgradeVariables(flags.applyPlanFlags.kubeConfigPath, flags.applyPlanFlags.cfgPath, flags.dryRun, userVersion)
+	if err != nil {
+		return err
+	}
+
+	ignorePreflightErrorsSet, err := validation.ValidateIgnorePreflightErrors(flags.applyPlanFlags.ignorePreflightErrors, cfg.NodeRegistration.IgnorePreflightErrors)
+	if err != nil {
+		return err
+	}
+
+	// Also set the union of pre-flight errors to InitConfiguration, to provide a consistent view of the runtime configuration:
+	cfg.NodeRegistration.IgnorePreflightErrors = ignorePreflightErrorsSet.List()
+
+	klog.V(1).Infoln("[upgrade/apply] verifying health of cluster")
```

Comment:
this should be printed above the call to: `CheckClusterHealth()`
changed as `klog.V(1).Infoln("[upgrade/health] verifying health of cluster")`

or removed.